tests/test_e2e_10_mef_eline.py

```diff
@@ -613,3 +696,550 @@ def test_080_create_and_remove_ten_circuits_ten_times(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        payload = {
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:2"
+            }
+        }
+
+        # It sets a new interface_id
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 200
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['uni_a']['interface_id'] == "00:00:00:00:00:00:00:01:2"
+
+    # Error (Patch returns: uni_z can't be be updated.)
+    @pytest.mark.xfail
+    def test_100_patch_evc_new_unis(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        payload = {
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:2"
+            }
+        }
+
+        # It sets a new interface_id
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 200
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['uni_z']['interface_id'] == "00:00:00:00:00:00:00:02:2"
+
+    def test_105_patch_start_date(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        start_delay = 60
+        start = datetime.now() + timedelta(minutes=start_delay)
+
+        payload = {
+            "start_date": start.strftime(TIME_FMT)
+        }
+
+        # It sets a new circuit's start_date
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 200
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['start_date'] == start.strftime(TIME_FMT)
+
+    def test_110_patch_end_date(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        end_delay = 60
+        end_date = datetime.now() + timedelta(minutes=end_delay)
+
+        payload = {
+            "end_date": end_date.strftime(TIME_FMT)
+        }
+
+        # It sets a new circuit's end_date
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['end_date'] == end_date.strftime(TIME_FMT)
```

Comment:
Here it would be interesting to have another assertion to verify if after waiting the end time, the circuit will be deactivated.

Reply:
This is resolved with commit [3678845](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/36788455fd3ef60027b1fd42a73568c0d1eb51cc)